File: 1017/价格查询.pas
var
 n:longint;
begin
 read(n);
 writeln('[1] apples');
 writeln('[2] pears');
 writeln('[3] oranges');
 writeln('[4] grapes');
 writeln('[0] Exit');
 if (n<>0) then write('price=');
 if (n=1) then write('3.0')
 else if (n=2) then write('2.5')
 else if (n=3) then write('4.1')
 else if (n=4) then write('10.2')
 else if (n>4) then write('0');
end.

File: 1017/价格查询.cpp
#include<iostream>
#include<cstdio>
using namespace std;
int main(){
	int n;
	double apples=3.00;
	double pears=2.50;
	double oranges=4.10;
	double grapes=10.20; 
	cout<<"[1] apples"<<endl
		<<"[2] pears"<<endl
		<<"[3] oranges"<<endl
		<<"[4] grapes"<<endl
		<<"[0] Exit"<<endl;
	cin>>n;
	if(n==1)
		printf("price=%.1f",apples);
	else if(n==2)
		printf("price=%.1f",pears);
	else if(n==3)
		printf("price=%.1f",oranges);
	else if(n==4)
		printf("price=%.1f",grapes);
	else if(n==0){
	}
	else{
		cout<<"price=0";
	}
		
		
	return 0;
}
